docs/ble.md

```diff
@@ -58,36 +73,36 @@ dictionary CharacteristicInit {
     NotifyCallback onNotify;            // optional
 };
 
-callback ReadCallback = void (unsigned long offset, FulfillReadCallback);
+interface Characteristic {
+    attribute ReadCallback onReadRequest;
+    attribute WriteCallback onWriteRequest;
+    attribute SubscribeCallback onSubscribe;
+    attribute UnsubscribeCallback onUnsubscribe;
+    attribute NotifyCallback onNotify;
+	attribute CharacteristicResult response;
+};
+
+callback ReadCallback = void (unsigned long offset,
+                              FulfillReadCallback fulfillreadcallback);
```

Comment:
This is why I'm getting confused - here `FulfillReadCallback` is treated like a type. So maybe in my earlier comment `ReadCallback` was the "type" and the variable should just be `callback`?

Reply:
As many ex-girlfriends have said to me: it's not you, it's me. :-)

The confounding inconsistency has been fixed (see above comment for more).